src/Resources/config/services.xml

```diff
@@ -22,6 +22,12 @@
             <tag name="console.command" />
         </service>
 
+        <service id="webgriffe_sylius_akeneo.command.clear" class="Webgriffe\SyliusAkeneoPlugin\Command\QueueCleanupCommand">
```

Comment:
You should rename the service id too, something like `webgriffe_sylius_akeneo.command.queue_cleanup` should be fine.